edgedb/base_client.py

```diff
@@ -287,6 +287,25 @@ async def _execute(self, execute_context: abstract.ExecuteContext) -> None:
                 ),
             )
 
+    async def describe(
+        self, describe_context: abstract.DescribeContext
+    ) -> abstract.DescribeResult:
+        cardinality, in_dc, out_dc, _ = await self._protocol._parse(
+            describe_context.query,
+            reg=protocol.CodecsRegistry(),
```

Comment:
`describe()` has no codecs cache for now

Reply:
That's OK